source/uri-options/uri-options.rst

```diff
@@ -280,6 +284,11 @@ pertaining to URI options apply here.
      - no
      - Relax TLS constraints as much as possible (e.g. allowing invalid certificates or hostname mismatches); drivers must document the exact constraints which are relaxed by this option being true
 
+   * - tlsDisableOcspEndpointCheck
+     - "true" or "false"
+     - false i.e. driver will reach out to OCSP endpoints `if needed <../ocsp-support/ocsp-support.rst#id1>`__.
```

Comment:
I admit this link is slightly fragile: I was unable to get a cross-document `:ref`` to work, and that section doesn't have a named id because there are two sections with the same name at different sublevels